coalib/bearlib/languages/Language.py

```diff
@@ -39,6 +39,7 @@ def parse_lang_str(string):
     versions = list(map(convert_int_float, str_versions))
     try:
         name, version = name.rsplit(maxsplit=1)
+        name = "".join(name.split())
```

Comment:
Hmm no that doesn't do it either.. You must remove white-space from the internal language string being compared... Amending the doc-string doesn't fix.